payments/schema/mutations.py

```diff
@@ -584,6 +586,70 @@ def mutate_and_get_payload(cls, root, info, **input):
         return DeleteOrderLineMutation()
 
 
+class SendBerthSwitchOfferMutation(graphene.ClientIDMutation):
+    class Input:
+        offers = graphene.List(graphene.NonNull(graphene.ID))
+        due_date = graphene.Date(description="Defaults to the Offer due date")
+        profile_token = graphene.String(
+            required=False, description="API token for Helsinki profile GraphQL API",
+        )
+
+    sent_offers = graphene.List(graphene.NonNull(graphene.ID), required=True)
+    failed_offers = graphene.List(graphene.NonNull(FailedOfferType), required=True)
+
+    @classmethod
+    @change_permission_required(BerthSwitchOffer,)
+    def mutate_and_get_payload(cls, root, info, offers, **input):
+        due_date = input.pop("due_date", None)
+
+        failed_offers = []
+        sent_offers = []
+
+        profile_token = input.get("profile_token")
```

Comment:
```suggestion
```

Reply:
profile_token is used in call to update_from_profile, should not be removed?